quipucords/tests/integration/test_smoker.py

```diff
@@ -206,17 +219,27 @@ def test_deployments_report(  # noqa: PLR0913
         assert target_fingerprints == expected_fingerprints
 
     def test_compare_standalone_json_reports_with_tarball(
-        self, django_client, report_id
+        self, client_logged_in, report_id
     ):
         """Compare individual reports with the ones bundled on the tarball endpoint."""
-        details_response = django_client.get(f"reports/{report_id}/details/")
-        assert details_response.ok, details_response.json()
+        details_response = client_logged_in.get(
+            reverse("v1:reports-details", args=(report_id,))
+        )
+        assert (
+            details_response.status_code == status.HTTP_200_OK
+        ), details_response.json()
```

Comment:
@infinitewarp I'm almost convincing myself to hack django client response so we can keep `response.ok`. it was so elegant... look at this mess! 🤣